src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+/* stylelint-disable string-quotes */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable color-hex-case */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  font-family: 'Roboto';
+  padding: 0 50px;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link:hover {
+  color: #00ACDC;
+}
+
+.is-active:hover {
+  color: #00ACDC;
+  text-decoration: none;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.nav__link {
+  text-decoration: none;
+  color: #000;
+
+  text-transform: uppercase;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+  padding: 23px 0;
+  display: block;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  background-color: #00ACDC;
+  left: 0;
+  border-radius: 1rem;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.is-active {
+  position: relative;
+  color: #00ACDC;
+
+}
+
+/* stylelint-disable-next-line block-no-empty */
+
+.body {
+  margin: 0;
+}
+
+.nav__list-item:last-of-type {
+  margin-right: 0;
+}
+
+/* stylelint-disable-next-line no-duplicate-selectors */
+/* stylelint-disable-next-line no-descending-specificity */
+.nav__list-item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
```

Comment:
you can reset default margin, padding once for all elements that need it:
body,
ul,
p {
  margin: 0;
  padding: 0;
}